tests/testthat/test-Logger.R

```diff
@@ -215,10 +215,10 @@ test_that("Logger sets log_file to NULL in DB if not writing to file", {
     expect_length(db_log_file, 1)
     expect_match(db_log_file, "^.+$")
 
-    logger$finalize()
 
+    connection_clean_up(conn)
```

Comment:
Shouldn't this be called after logger$finalize() as the cleanup closes the connection and then finalize does not pass the initial if statement